amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/table/TableService.java

```diff
@@ -146,4 +147,8 @@ Blocker block(
    * @return block list
    */
   List<Blocker> getBlockers(TableIdentifier tableIdentifier);
+
+  List<TableRuntimeBean> getTableRuntimesForAllGroup(int limit, int offset);
```

Comment:
I think TableService could be eliminated from DashboardServer in the future, or depend a service implementation, not a POJO

So I don't suggest this way